engine/src/main/resources/assets/i18n/menu.lang

```diff
@@ -31,6 +31,7 @@
     "binding-nui-editor": "binding-nui-editor",
     "binding-nui-skin-editor": "binding-nui-skin-editor",
     "binding-pause": "binding-pause",
+    "binding-screenshot": "binding-screenshot",
```

Comment:
This should be moved one line down so that the whole file remains arranged in alphabetical order.